assets/src/js/external_links.js

```diff
@@ -18,7 +18,12 @@ export const setupExternalLinks = () => {
     }
 
     link.target = link.target ? link.target : '';
-    link.classList.add('external-link');
+    if (!commonExternalDomains.includes(link.hostname)){
+      link.classList.add('external-link');
+    }
+    else {
```

Comment:
External links were getting _blank, but only the commonly used external domains, that indeed doesn't seem right :smile: 
At least solved that -for now- in [`874a3e4` (#1746)](https://github.com/greenpeace/planet4-master-theme/pull/1746/commits/874a3e41dd0d1da4c0625f4022633903cc5ab897).

I'm doubting though.. Should commonly used domains maybe NOT be opened in _blank? ie GPNL uses an external domain for petitions, it might be beneficial to open that one in the same tab. Kinda edge casey though, so maybe not important for the generic case?

Reply:
> I'm doubting though.. Should commonly used domains maybe NOT be opened in _blank? ie GPNL uses an external domain for petitions, it might be beneficial to open that one in the same tab. Kinda edge casey though, so maybe not important for the generic case?  

So I'm realizing now that your PR says 

> These ("Commonly used") domains should not get the external link icon and **should always open in a new tab**

but the [ticket](https://github.com/greenpeace/planet4/issues/174) says
> Help text: "Links for these domains will **not open in a new tab** and will hide the external link icon."
> Adjust external link behaviour to **remove external link icon and opening in a new tab for these domains**.

There might be a misunderstanding then ? From what I understand, links on Commonly used domains should open in the same tab.

Your last change ends up in a double assignement for [`link.target`](https://github.com/greenpeace/planet4-master-theme/pull/1746/files/c65b866f79088cbef36f3a351db74ad161c61dae..874a3e41dd0d1da4c0625f4022633903cc5ab897#diff-33892ae9bc87d6339a56a69f3d9810a8da1f4fcc306acace491add4ff76e80aaR20-R21), I think this should work
```js
if (commonExternalDomains.includes(link.hostname)) {
  // Commonly used domains: force open in same tab, no external link icon
  link.target = '';
} else {
  // Other domains: follow post config, add external link icon
  link.target = link.target ?? ''; // follow post config
  link.classList.add('external-link'); // external link icon
}
```